styles/styles.css

```diff
@@ -69,6 +69,10 @@
   --color-footer-gray: #a3a3a3;
   --box-shadow-footer-top-gray: #0e1f354d;
   --veritcal-line-footer-top-gray: #9797974d;
+  --box-shadow-trending-news: #97979733;
```

Comment:
please see if we can rename these to generic names irrespective of the blocks, as color names

Reply:
done